tests/unit/utils/test_app_utils.py

```diff
@@ -0,0 +1,9 @@
+"""Unit Test for the Utils class."""
+from ols.app.utils import Utils
+
+
+def test_get_suid():
+    """Test get_suid method."""
+    suid = Utils.get_suid()
+    assert len(suid) == 32
+    assert isinstance(suid, str)
```

Comment:
in theory you might test it using regexp

Reply:
yes

Reply:
pattern check has been added now using the pattern `[a-f0-9]{32}`.
The test for length is remove as the pattern already checks the length of 32 characters.